src/test/java/lt/andriaus/hangman/usecase/GameInRamManagerIntegrationTest.java

```diff
@@ -0,0 +1,113 @@
+package lt.andriaus.hangman.usecase;
+
+import lt.andriaus.hangman.database.Database;
+import lt.andriaus.hangman.domain.Game;
+import lt.andriaus.hangman.domain.GameException;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@ExtendWith(MockitoExtension.class)
+class GameInRamManagerIntegrationTest {
+    private static GameManager gameManager;
+
+
+    @BeforeAll
+    static void setup(){
+        Database<String> wordDB = new RamDatabase<>();
+        wordDB.save("Hello");
+        wordDB.save("Adele");
+        Database<Game> gameDB = new RamDatabase<>();
+        gameManager = new GameInRamManager(wordDB, gameDB);
+    }
+
+    @Test
+    void shouldCreateGame() {
+        int newGameId = gameManager.createGame();
+        assertThat(newGameId).isGreaterThan(-1);
+    }
+
+    @Test
+    void shouldCreateAndLoadGame() {
+        int newGameId = gameManager.createGame();
+        Optional<Game> newGame = gameManager.loadGame(newGameId);
+        assertThat(newGame.isPresent()).isTrue();
+        assertThat(Arrays.asList(new String[]{"HELLO", "ADELE"}).contains(newGame.get().getWord())).isTrue();
```

Comment:
`Arrays.asList(new String[]{"HELLO", "ADELE"}` -> `List.of("HELLO", "ADELE")`

Reply:
šiaip dažniausiai nori kontroliuti tokį dalyką, tai arba db palikti vieną žodį, arba pakišti fake "random" generatorių vietoj įhardcodinto implementacijoj